forge-gui/res/cardsfolder/upcoming/electroduplicate.txt

```diff
@@ -0,0 +1,8 @@
+Name:Electroduplicate
+ManaCost:2 R
+Types:Sorcery
+K:Flashback:2 R R
+A:SP$ CopyPermanent | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | AddKeywords$ Haste | AddSVars$ TrigEOT | SpellDescription$ Create a token that's a copy of target creature you control, except it has haste and "At the beginning of the end step, sacrifice this token."

```

Comment:
trigger missing

Reply:
Isn't it just after this line?

Reply:
it's not an activated ability, so you should use AddTriggers